custom_components/securitas/alarm_control_panel.py

```diff
@@ -63,15 +63,6 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
             )
     add_entities(alarms)
 
-
-def set_arm_state(state, code=None):
```

Comment:
It seems this function is not used anywhere, the implementation in the class is used instead

Reply:
Yes, you are right